Xamarin.Forms.Platform.Android/CollectionView/ObservableGroupedSource.cs

```diff
@@ -2,19 +2,20 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Collections.Specialized;
+using System.Threading;
 
 namespace Xamarin.Forms.Platform.Android
 {
 	internal class ObservableGroupedSource : IGroupableItemsViewSource, ICollectionChangedNotifier
 	{
 		readonly ICollectionChangedNotifier _notifier;
 		readonly IList _groupSource;
+		readonly SynchronizationContext _synchronizationContext;
```

Comment:
Is there a reason you went with custom code here instead of using the cross-platform Dispatcher stuff?

Reply:
Replaced the usage of `SynchronizationContext` by our cross-platform Dispatcher.